python/rpdk/java/templates/init/guided_aws/StubDeleteHandler.java

```diff
@@ -105,6 +105,10 @@
                         logger.log(String.format("%s [%s] deletion has stabilized: %s", ResourceModel.TYPE_NAME, model.getPrimaryIdentifier(), stabilized));
                         return stabilized;
                     })
-                    .success());
+                    .progress()
+            )
+
+            // STEP 3 [TODO: return the successful progress event without resource model]
+            .then(progress -> ProgressEvent.defaultSuccessHandler(null));
```

Comment:
lets return a progress event without the model.

Reply:
`ProgressEvent.defaultSuccessHandler(null)` returns without a model